charts/home-assistant/values.yaml

```diff
@@ -1,315 +1,108 @@
-    # bearerTokenSecret:
-    # Relabel metrics if needed example removes pod and instance labels from metrics beginning with hass
-    # metricRelabelings: []
-    #  - regex: hass.*
-    #    replacement: ""
-    #    sourceLabels:
-    #     - __name__
-    #    targetLabel: pod
-    #  - regex: hass_.*
-    #    replacement: ""
-    #    sourceLabels:
-    #     - __name__
-    #    targetLabel: instance
-
-vscode:
-  enabled: false
-
-  ## code-server container image
-  ##
-  image:
-    repository: codercom/code-server
-    tag: 3.7.2
-    pullPolicy: IfNotPresent
-
-  ## VSCode password
-  # password:
-
-  ## path where the home assistant configuration is stored
-  hassConfig: /config
-
-  ## path where the VS Code data should reside
-  vscodePath: /config/.vscode
-
-  ## Additional hass-vscode container environment variable
-  ## For instance to add a http_proxy
-  ##
-  extraEnv: {}
-
-  ## Set to "-" to use default argument list
-  ## Otherwise convert to list of arguments
-  args: "-"
-
-  ingress:
-    enabled: false
-    annotations: {}
-      # kubernetes.io/ingress.class: nginx
-      # kubernetes.io/tls-acme: "true"
-    path: /
-    hosts:
-      - home-assistant.local
-    tls: []
-    #  - secretName: home-assistant-tls
-    #    hosts:
-    #      - home-assistant.local
-
-  service:
-    type: ClusterIP
-    port: 80
-    annotations: {}
-    labels: {}
-    clusterIP: ""
-    ## List of IP addresses at which the hass-vscode service is available
-    ## Ref: https://kubernetes.io/docs/user-guide/services/#external-ips
-    ##
-    externalIPs: []
-    loadBalancerIP: ""
-    loadBalancerSourceRanges: []
-    # nodePort: 30000
-
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #  cpu: 100m
-  #  memory: 128Mi
-  # requests:
-  #  cpu: 100m
-  #  memory: 128Mi
-
-nodeSelector: {}
-
-tolerations: []
-
-affinity: {}
-
-podAnnotations: {}
-
-# Any extra volumes to define for the pod
-extraVolumes: []
-  # - name: example-name
-  #   hostPath:
-  #     path: /path/on/host
-  #     type: DirectoryOrCreate
-
-# Any extra volume mounts to define for the containers
-extraVolumeMounts: []
-#   - name: example-name
-#     mountPath: /path/in/container
-
-# Enabled mariadb
-# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/mariadb
-mariadb:
-  enabled: false
-  architecture: standalone
-  auth:
-    database: home-assistant
-    username: home-assistant
-    password: home-assistant-pass
-    rootPassword: home-assistantrootpass
-  primary:
-    persistence:
-      enabled: false
-      # storageClass: ""
-
-# Enabled postgres
```

Comment:
Did you remove this on purpose? Because then I think we should also remove the dependency in `Chart.yaml`. Same thing for mariadb / influxdb. 

I would propose to leave the database support in the values.yaml so users know what their options are. 
Unless want to move them out of the chart so they have to be deployed separately?

Reply:
This was a mistake :O Good catch!